contracts/interfaces/base/IModuleManagerEventsAndErrors.sol

```diff
@@ -31,8 +31,8 @@ interface IModuleManagerEventsAndErrors {
     /// @param module The address of the uninstalled module.
     event ModuleUninstalled(uint256 moduleTypeId, address module);
 
-    /// @dev Thrown when an attempt is made to uninstall the last validator module, which is prohibited.
-    error CannotRemoveLastValidator();
+    /// @notice Thrown when no validators exist or when attempting to remove the last one.
+    error MissingValidator();
```

Comment:
why this is renamed?
new one is less readable / obvious

Reply:
suggest new name ! 
We check if there is a validator on initialization based on #15 , so using CannotRemoveLastValidator while initializing the account doesn't make sense to me

Reply:
again you linked the wrong link

Reply:
Im not linking any link, Github does it automatically if you do `# + number`, check # 15 on the description

Reply:
ok lol then paste full cantina url

Reply:
check comment above.